packages/neon_framework/test/option_test.dart

```diff
@@ -282,7 +287,7 @@ void main() {
     });
 
     test('Disable', () {
-      final callback = MockCallbackFunction();
+      final callback = MockCallbackFunction<void>();
```

Comment:
it is similar here:
Do we want to test that it works with a void method or any arbitrary method? Yes we can use `dynamic` as a generic but I think just ignoring the rule makes more sense.